### PR TITLE
Ipv6 nf test handling and demonstrate accepted ipv4 and 6 packets in a chained nf.

### PR DESCRIPTION
**Pull request by zeeshanlakhani**

### CODE REVIEW

File: test/ipv4or6/check.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+TEST_NAME=tcp-payload
```
[the remark this answers is not in the document]
Comment (zeeshanlakhani) [reply test/ipv4or6/check.sh]: Lol. Yeah. Will fix upon return to nyc in an hour.

**Comment by zeeshanlakhani** [comment]
@drunkirishcoder think we can assert that within the nf/test itself.

**Comment by zeeshanlakhani** [comment]
figured out how to do that, but didn't feel the need here.

**Comment by drunkirishcoder** [comment]
interesting, ok.

File: test/ipv4or6/src/main.rs [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+                                now - start,
+                                rx_pkts as f64 / (now - start),
+                                (pkts.1 - pkts_so_far.1) as f64 / (now - start)
+                            );
+                            last_printed = now;
+                            start = now;
+                            pkts_so_far = pkts;
+                        }
+                    }
+                }
```
Comment (clinedome) [inline test/ipv4or6/src/main.rs]: nbd, but we can lose this whole big chunk if we just put a default dur on there. If we think it'd be worthwhile to set up a "listen forever and just handle/dump whatever comes through" though, I'm fine with leaving all this.

Comment (zeeshanlakhani) [reply test/ipv4or6/src/main.rs]: sure.

**Comment by zeeshanlakhani** [comment]
@clinedome gonna make it similar to macswap, in that we either give it a dur or it loops waiting for packets (and therefore can be used as an nf chain for integrated work and examples).

**Comment by clinedome** [comment]
@zeeshanlakhani perfect :+1:

**Comment by zeeshanlakhani** [comment]
@seancribbs did you want to give this one last looksee?

**Pull request merged**